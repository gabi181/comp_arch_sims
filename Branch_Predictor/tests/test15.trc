1 7 17 0 local_history global_tables using_share_mid
0x4C7390EC N 0x3C64B8D0
0x74747BC4 T 0x6AED8154
0x4BC2E9C8 T 0x52844408
0x74747BC4 T 0x6AED8154
0x1CCEE9A0 T 0x02E2C08C
0x4C7390EC T 0x3C64B8D0
0x74747BC4 T 0x6AED8154
0x4BC2E9C8 T 0x52844408
0x4AFC7B98 T 0x681E4CEC
0x4C7390EC N 0x3C64B8D0
0x74747BC4 N 0x6AED8154
0x4AFC7B98 T 0x681E4CEC
0x1CCEE9A0 T 0x02E2C08C
0x1CCEE9A0 T 0x02E2C08C
0x1CCEE9A0 T 0x02E2C08C
0x4AFC7B98 N 0x681E4CEC
0x4AFC7B98 N 0x681E4CEC
0x1CCEE9A0 N 0x02E2C08C
0x4C7390EC N 0x3C64B8D0
