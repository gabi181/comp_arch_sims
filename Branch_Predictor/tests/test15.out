0x4c7390ec N 0x4c7390f0
0x74747bc4 N 0x74747bc8
0x4bc2e9c8 N 0x4bc2e9cc
0x74747bc4 N 0x74747bc8
0x1ccee9a0 N 0x1ccee9a4
0x4c7390ec N 0x4c7390f0
0x74747bc4 N 0x74747bc8
0x4bc2e9c8 N 0x4bc2e9cc
0x4afc7b98 N 0x4afc7b9c
0x4c7390ec N 0x4c7390f0
0x74747bc4 N 0x74747bc8
0x4afc7b98 N 0x4afc7b9c
0x1ccee9a0 N 0x1ccee9a4
0x1ccee9a0 N 0x1ccee9a4
0x1ccee9a0 N 0x1ccee9a4
0x4afc7b98 N 0x4afc7b9c
0x4afc7b98 N 0x4afc7b9c
0x1ccee9a0 N 0x1ccee9a4
0x4c7390ec N 0x4c7390f0
flush_num: 12, br_num: 19, size: 311b
